Makefile

```
@@ -1,9 +1,12 @@
 .DEFAULT_GOAL := default
 
 lint:
-	docker-compose run --rm lint
+	docker run -it --rm -v `pwd`:/plugin:ro buildkite/plugin-tester
```

Nit: (avoids spawning a shell and process just to get the working dir…)

```suggestion
	docker run -it --rm -v "$PWD":/plugin:ro buildkite/plugin-tester
```